Libraries/Opc.Ua.Configuration/ApplicationInstance.cs

```diff
@@ -320,15 +332,12 @@ public async Task<ApplicationConfiguration> LoadApplicationConfiguration(bool si
             ushort lifeTimeInMonths)
         {
             Utils.Trace(Utils.TraceMasks.Information, "Checking application instance certificate.");
-
-            ApplicationConfiguration configuration = null;
-
             if (m_applicationConfiguration == null)
             {
-                await LoadApplicationConfiguration(silent);
+                await LoadApplicationConfiguration(silent).ConfigureAwait(false);
```

Comment:
Method name should end with Async. Also, consider changing name to explain difference between this and method "LoadAppConfig" or create an overload.

Reply:
someday hopefully LoadApplication can be improved when there are more breaking changes...